### ISSUE TITLE
Object VerificationLevel is missing

### ISSUE THREAD

**Issue by tabbertpeter** [PMSFIT/SSPTraceability#16]
**Description:** There is an object ValidationLevel in the schema, but a corresponding object VerificationLevel is missing. This should be added.
**Rationale:** System-X originally suggested both objects. However, only ValidationLevel was adopted.
A representative of Daimler AG has also pointed out that VerificationLevel is required.

**Comment by pmai** [comment]
I would venture that this only makes sense if proper definitions of validation vs. verification at the object level can be agreed upon. Otherwise this will be misused, just as validation vs. verification is currently very often misused (also take into account whether the level can actually be achieved at the object level under discussion, or is not e.g. a system-level concept).

**Comment by tabbertpeter** [comment]
I agree to Pierre's comment or me also a clear distinction of validation and verification is essential, Also the basis of both is crucial (verified/validate against what?). Nevertheless, considering both might be usefull

**Comment by DrCaFr** [comment]
SMMD refers to "NASA-STD-7009". 
In my opinion, this standard is in itself inconsistent wrt. to the definition of "validation". 
It states on p. 12: "Domain of Validation: The region enclosing all sets of model inputs for which the M&S’s responses compare favorably with the referent.",
but on p. 15: "Validation: The process of determining the degree to which a model or a simulation is an accurate representation of the real world from the perspective of the intended uses of the M&S.".
See that "real world" is referenced by the 2nd definition, only. 

According to my perception, the notion of "verification" is generally associated with (hopefully) exactly formulated requirements, whereas "validation" refers more to some comparison with "reality" (frequently missing a definition about what "reality" actually is) or with value for some user (group). 

**Conclusion:** I regard "verification" to be much more appropriate to numerical simulation than "validation".
